checks/networking/dcos_net_errors.go

```diff
@@ -0,0 +1,76 @@
+package errors
+
+import (
+	"bufio"
+	"fmt"
+	"regexp"
+	"github.com/adyatlov/bun"
+)
+// http://erlang.org/doc/apps/kernel/logger_chapter.html#log_level
+var errorsRegex = "\\[error\\]|\\[critical\\]|\\[alert\\]|\\[emergency\\]"
+
+func init() {
+	builder := bun.CheckBuilder{
+		Name: "net-errors-checker",
+		Description: "Identify errors on dcos-net logs",
+ 		CollectFromMasters:      collect,
+		CollectFromAgents:       collect,
+		CollectFromPublicAgents: collect,
+		Aggregate:               aggregate,
+	}
+	check := builder.Build()
+	bun.RegisterCheck(check)
+}
+
+func collect(host bun.Host) (ok bool, details interface{}, err error) {
+	errorMatcher, _ := regexp.Compile(errorsRegex)
+  keys := 0
+
+	file, err1 := host.OpenFile("net")
```

Comment:
Is it intentionally non-configurable?

Reply:
There is a file who maps "net" to the actual diagnostics bundle file `filetypes/files_type_yaml.go`. In what sense you want this to be configured? A list of files with a single one maybe?